custom/icds/management/commands/copy_icds_app_2_of_2.py

```diff
@@ -0,0 +1,66 @@
+import sys
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.app_manager.suite_xml.post_process.resources import copy_xform_resource_overrides
+from corehq.apps.linked_domain.applications import get_master_app_by_version
+
+
+class Command(BaseCommand):
+    help = """
+        For use when creating a new master app for ICDS.
+
+        After running copy_icds_app_1_of_2 on the master domain to create a copy of the master app,
+        run this command on all apps linked to that master app.
+
+        Looks up any ResourceOverrides for the given linked app that are tied to the original
+        master app's form ids. For any such override, the script creates a new ResourceOverride
+        where the pre_id is the unique id of that same form in the master app copy and the
+        post_id is the same as the original override.
+
+        This command is necessary until the ICDS apps are pinned to 2.47.4+.
```

Comment:
just curious, what does pinned to 2.47.4+ imply?

Reply:
It fixes the bug where forms crash on mobile if their unique id gets changed. If everyone were on 2.47.4+, we wouldn't need the `ResourceOverride` code at all.

Reply:
okay thanks
so it's simply `until the ICDS users are on 2.47.4+.`?
When i read pinned, i thought we were restricting users to some version.

Reply:
It is indeed just `until the ICDS users are on 2.47.4+`, but the way we would know/enforce that all users are on 2.47.4+ is by restricting the version, right?

Reply:
you mean all users are on commcare version `2.47.4` or more.
I don't think there is a way to be sure about that ever. 
What do you mean by `restricting the version`?

Reply:
By restricting, I mean only allow users to update to the next AWW version if they're on 2.47.4. I don't remember what mechanism we have for this - prompted updates, maybe? - but I'm certain we have one.

Reply:
i guess something related to setting the app version in the app.
and even in that, not clear what the message phone would complain about and if user would actually make an effort to update.
If you are clear about it, fine to abandon this conversation.

Reply:
Oh, you're right, it's just a matter of setting the CC version in the app. I think we're okay to drop this conversation.